### DISCUSSION TITLE
How to set already stored html programmatically without using autofocus plugin or manual focus

### DISCUSSION THREAD

**Discussion by TanayWagh01** [facebook/lexical#3988] [Q&A]
How do I set the already stored HTML at initial mount without using AutoFocus Plugin?
 
 I want to set the Lexical Editor state ( using already stored HTML ) programatically by clicking on a button. The issue I am facing is with setting this initial value. Unless I focus on editor or use AutoFocus plugin, the Editor State doesn't get updated.

If I use AutoFocus plugin, the webpage automatically scrolls down to RichText editor and gets focused which is creating bad user experience.

How do I set initial value in above case without using AutoFocus Plugin?

**Comment by ricardosantoyo93** [comment]
I am facing this same issue right now. Commenting here to raise awareness, and to see if there is a workaround we can do in the meantime.
